_posts/2025-03-26-kubeflow-1.10-release.md

```diff
@@ -0,0 +1,276 @@
+performanace improvements to the existing pull based design. For further details, please refer to the
+[Push-Based Metrics Collection blog post](https://blog.kubeflow.org/gsoc-2024-project-6/).
+
+## Dashboard & Notebooks
+
+Kubeflow 1.10 improves the observability and usability of Notebooks, while providing updated
+[default images](https://github.com/kubeflow/kubeflow/pull/7687).
+
+### Prometheus Metrics for Notebooks
+
+Both the Notebooks component and CRUD backends now feature Prometheus metrics. Notebooks expose custom metrics using the
+prom-client library, and CRUD backends utilize the prometheus_flask_exporter library. This ensures consistent metrics
+integration across all backend services.
+
+### More Descriptive Error Messages
+
+Error messages for notebook creation failures due to resource constraints are now more descriptive. Users can quickly
+identify issues such as insufficient resources.
+
+## Spark Operator
+
+The Spark Operator, now integrated as a core Kubeflow component, includes several key enhancements focusing on
+architecture, security, and performance:
+
+* Rebuilt with Controller Runtime (v2.0.0): Modernized core architecture using controller-runtime, aligning with
+  Kubernetes controller patterns for improved structure, extensibility, and testability.
+* YuniKorn Gang Scheduling Support (v2.0.0): Enables efficient scheduling of Spark driver & executor pods as a group,
+  ideal for large-scale data pipelines with resource guarantees.
+* Enhanced Security Contexts & SeccompProfile Support (v2.1.1): Adds support for `seccompProfile: RuntimeDefault` & 
+  `readOnlyRootFilesystem`, aligning with Kubernetes Pod Security Standards and minimizing security risk.
+
+## KServe
+
+KServe v0.14.1 introduces several essential features that enhance its capabilities for deploying and managing machine
+learning models.
+
+### New Python SDK
+
+The release includes a new Python SDK with both REST and GRPC inference clients, offering asynchronous support and the
+ability to handle tensor data in binary format.
+
+### OCI Storage for Models
+
+OCI storage for models has also been promoted to a stable feature, with improvements to stability by configuring OCI
+models as init containers.
+
+### Model Cache Feature
+
+Additionally, the introduction of the Model Cache feature leverages local node storage to reduce model load times,
+especially for large models, enhancing scalability.
+
+### Hugging Face Integration
+
+KServe v0.14.1 further expands integration with Hugging Face, enabling direct model deployment from the Hugbing Face hub
+via a new `hf://` URI schema.
+
+## What comes next?
+
+If you want to take a peek into the Kubeflow 1.11 roadmap planning and contribute with your ideas, see
+[Notebooks](https://github.com/kubeflow/kubeflow/issues/7459),
+[Manifests & Security](https://github.com/kubeflow/manifests/milestone/2), Pipelines, Model Registry, Katib,
+Training Operator.
```

Comment:
here it seems we have a link to notebooks, manifests but not for the rest of the components.

Reply:
I couldn't find tracking issues for the other projects.